sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/instrumentation/NoopInstrumentationContext.java

```diff
@@ -1,36 +1,55 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package io.clientcore.core.instrumentation.tracing;
+package io.clientcore.core.implementation.instrumentation;
 
 import io.clientcore.core.instrumentation.InstrumentationContext;
+import io.clientcore.core.instrumentation.tracing.Span;
 
-final class NoopInstrumentationContext implements InstrumentationContext {
+/**
+ * No-op implementation of {@link InstrumentationContext}.
+ */
+public final class NoopInstrumentationContext implements InstrumentationContext {
     public static final NoopInstrumentationContext INSTANCE = new NoopInstrumentationContext();
 
     private NoopInstrumentationContext() {
     }
 
+    /**
```

Comment:
Just a nit: if an API is `@Override` it implicit uses inheritDoc, so these aren't explicitly needed